pkg/minikube/mount/cifs_windows.go

```diff
@@ -0,0 +1,159 @@
+// +build windows
+
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"github.com/danieljoos/wincred"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh/terminal"
+	"k8s.io/minikube/pkg/minikube/drivers/hyperv"
```

Comment:
Copy or move the hyperv functions into this package instead of making hyperv a dependency of mount.

Reply:
I am not clear on this. The Hyper-V related functions should be separate and seem to be in the correct place? The mount dependency on Hyper-V is only when it is built against Hyper-V. Could you please rephrase?